### ISSUE TITLE
关于SSD卷积代替全连接层的请教

### ISSUE THREAD

**Issue by junnyin** [YunYang1994/TensorFlow2.0-Examples#131]
`self.pool5   = tf.keras.layers.MaxPooling2D(3, strides=1, padding='same') # [19, 19, 512]

        # fc6, => vgg backbone is finished. now they are all SSD blocks
        self.fc6 = tf.keras.layers.Conv2D(1024, 3, dilation_rate=6, activation='relu', padding='same')
        # fc7
        self.fc7 = tf.keras.layers.Conv2D(1024, 1, activation='relu', padding='same')`
请问fc6为什么需要用到空洞卷积?
